doc/chapters/filesystem/index.rst

```diff
@@ -101,13 +101,21 @@ the ``lower`` directory. If a file from the ``lower`` filesystem is opened and
 then edited, it will be written in the ``upper`` directory and the user of the
 mountpoint will see only the changed file.
 
+.. note:: Enabling the write buffer will enable write buffers for all
+          filesystems, both the rootfs and the all bindmounted filesystems
+          inside the container. Also, the filesystems will no longer be
+          bindmounted technically, as bindmounting and overlayfs are mutually
+          exclusive.
```

Comment:
I believe so? The service tests do this since the bug story was fixed regarding the service tests?